taichi/codegen/metal/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# ./taichi/codegen/metal/CMakeLists.txt
+
+add_library(metal_codegen)
```

Comment:
I noticed that metal's shaders are spread in both `codegen/` and `runtime/`, so are OpenGL's. Since we plan to make `codegen` depend on `runtime`, we can place these shaders all under runtime.

https://github.com/taichi-dev/taichi/blob/b1ba4572a1c0fef12fe71b63c8e28b289c77e2ad/taichi/codegen/metal/codegen_metal.cpp#L31-L32